flaskbb/user/models.py

```diff
@@ -250,6 +251,7 @@ def check_password(self, password):
         return check_password_hash(self.password, password)
 
     @classmethod
+    @deprecated(_("Use authentication services instead."))
```

Comment:
I don't think that we have to translate this :)

Reply:
Maybe not. I'll yank it off to be sure.